src/Raven.Server/Documents/ETL/Providers/SQL/RelationalWriters/RelationalDatabaseWriter.cs

```diff
@@ -468,8 +471,43 @@ public static void SetParamValue(DbParameter colParam, SqlColumn column, List<Fu
                             if (objectValue.TryGetMember(nameof(SqlDocumentTransformer.VarcharFunctionCall.Type), out object dbType) &&
                                 objectValue.TryGetMember(nameof(SqlDocumentTransformer.VarcharFunctionCall.Value), out object fieldValue))
                             {
-                                colParam.DbType = (DbType)Enum.Parse(typeof(DbType), dbType.ToString(), false);
-                                colParam.Value = fieldValue.ToString();
+                                try
+                                {
+                                    colParam.DbType = (DbType)Enum.Parse(typeof(DbType), dbType.ToString(), false);
```

Comment:
Use TryParse?
BTW. Do we want to be case sensitive here?

Reply:
The `DbType' setter of OracleParameter throws, not the enum
![image](https://user-images.githubusercontent.com/16625387/110321969-17ff1f00-801b-11eb-88a9-4755147ce8b3.png)

Yes, this is a case when you pass the type explicitly